### ISSUE TITLE
Add possibility to unset a tag in the question catalog definition

### ISSUE THREAD

**Issue by wielandb** [OPENER-next/OpenStop#270]
### Idea

It would be great if it would be possible to define the answer to a question in a way that some tag gets removed if it is set.

### Implementation

The JSON could look like this:

```
      {
          "name": "Example answer",
          "osm_tags": {
            "exampleTag": "exampleValue",
            "tagThatShouldBeRemoved": None
          }
      },
```
Where a tag in `osm_tags` whose value is `None`/`null` will be removed if present.

**Comment by Robbendebiene** [comment]
I agree that this might be helpful in the future if we support "alternative answers". Because sometimes something got removed but the tag is still present causing OpenStop to ask an unanswerable question.

Currently any expression in the constructor returns `null` whenever an error occurs. It is then treated as a noop. Of course this could be changed so whenever we return `null` and a tag is present we remove the tag. Still I think it is worth considering alternatives.

Alternatives are
- empty string `""`
- bool: `false`
- reserved placeholders `{{my_placeholder}}`
- special constructor expression `[DELETE]`

**Comment by wielandb** [comment]
My favourite at this point would be `false`, because:

- `false` in a condition already means "not set" and thus the meaning transpires
- Using an empty string or reserved placeholders might work, but it always leaves some room for error/ambiguity
- A constructor expression would surely also work, but would be more complex
